build/scripts/tag-common.sh

```diff
@@ -63,6 +63,15 @@ function commit() {
   git commit --message "${commit_msg}"
 }
 
+function tag_and_push_release() {
+    local version="${1}"
+    local msg="${2}"
+    for tag in "${version}" "v${version}"; do
```

Comment:
I'm not sure about: `"${version}" "v${version}"` - should it be written twice?

Reply:
Yes, we need to push tags with and without the `v` prefix.